docs/user-guide/khaas-101.md

```diff
@@ -0,0 +1,134 @@
+# KubeHound as a Service (KHaaS)
+
+KHaaS enables you to use KubeHound in a distributive way. It is being splitted in 2 main categories:
+
+* The ingestor stack which includes the `graphdb`, `datadb`, `UI` and `grpc endpoint`.
+* The collector (the kubehound binary) which will dump and send the k8s resources to the KHaaS `grpc endpoint`.
+
+[![](../images/khaas-architecture.png)](../images/khaas-architecture.png)
+
+!!! Note
+    You need to deploy the data storage you want to use ([AWS s3 in our example](https://github.com/DataDog/KubeHound/tree/main/deployments/terraform)).
+
+## Deploying KHaaS - Ingestor stack
+
+### Docker deployment
+
+To run the KubeHound as a Service with `docker` just use the following [compose files](https://github.com/DataDog/KubeHound/tree/main/deployments/kubehound):
+
+```bash
+cd ./deployments/kubehound
+docker compose -f docker-compose.yaml -f docker-compose.release.yaml -f docker-compose.release.ingestor.yaml up -d
+```
+
+By default the endpoints are only exposed locally:
+
+* `127.0.0.1:9000` for ingestor endpoint.
+* `127.0.0.1:8888` for the UI.
+
+!!! Warning
```

Comment:
see [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)
```suggestion
> [!WARNING]
```

Reply:
nop, this is for the wiki, it is not the same. Run the `make local-wiki`